post/2017/11-12-speed-release.md

```diff
@@ -0,0 +1,62 @@
+---
+layout:     post
+title:      Inexor 0.9.0 "Speed Release" announcement
+date:       2017-11-12 20:00 +0100
+author:     a_teammate
+summary:    The Inexor Speed release is finally there.
+---
+
+Hello there everyone, we have big news!
+Inexor 0.9.0@beta is finally released!
+
+Yes it's still unstable software, the switch from "0.8.0-alpha" to "0.9.0@beta" is only a result of our most visible change for end-users:
+We are going rolling release!
+
+
+### Rolling-release?
+You've heard it right. Starting with v0.9 release, every closed pull request (that can be every week, or day, depending on how much people contribute) will automatically be a new release and people will automatically download it the next time they start Inexor.
+
+Users can choose to play with different versions of different channels in different instances of the game.
+That is why on GitHub the (still unstable) release *0.9.0* has the name *0.9.0@beta*: it is released in the beta channel.
```

Comment:
This paragraph maybe
```
Users can choose to play with different versions of different channels in different instances of the game.
With some slight changes in your config you could also add a different release provider. For example to update your favorite mod automatically when they get provided by a thirdparty-person.
Switching between versions is then instance dependent. You might want to test mods just briefly and you can do so by just changing the version and release channel.

We (the Inexor-Team) is currently providing releases in the `alpha` channel, thats why its @alpha and not -alpha.

This might be comparable to i.e. Windows updates in the `fast ring`, but in a more generically applicable way.